institutions-api/src/test/resources/application-createschema.conf

```diff
@@ -44,6 +44,8 @@ cassandra-snapshot-store {
 hmda {
   institution {
     createSchema = false
+    yearsAvailable = "2018,2019,2020,2021,2022"
+    defaultYear = "2022"
```

Comment:
Good for now, gonna start creating tech debt issues on configurability so we don't have to keep rebuilding and redeploying the image.